packages/server/src/__tests__/documentStore.test.ts

```diff
@@ -28,48 +28,40 @@ const documentNodeMatcher = {
   },
 };
 
+const hash = 'ec2e01311ab3b02f3d8c8c712f9e579356d332cd007ac4c1ea5df727f482f05f';
 const operations = {
   simple: {
     op: { query: 'query { hello }' },
-    hash: 'ec2e01311ab3b02f3d8c8c712f9e579356d332cd007ac4c1ea5df727f482f05f',
+    hash,
   },
 };
 
 describe('ApolloServer documentStore', () => {
   it('documentStore - undefined', async () => {
-    const server = new ApolloServer<BaseContext>({
+    const server = new ApolloServer({
       typeDefs,
       resolvers,
     });
 
     await server.start();
 
     // Use [] syntax to access a private method.
-    const { documentStore } = (
-      await server['_ensureStarted']()
-    ).schemaManager.getSchemaDerivedData();
-    expect(documentStore).toBeInstanceOf(InMemoryLRUCache);
-    const embeddedStore = documentStore as InMemoryLRUCache<DocumentNode>;
+    const { schemaManager } = await server['_ensureStarted']();
+    const { documentStore } = schemaManager.getSchemaDerivedData();
+    assert(documentStore);
+    expect(documentStore).toBeInstanceOf(Keyv);
 
     await server.executeOperation(operations.simple.op);
 
-    expect(await embeddedStore.getTotalSize()).toBe(403);
-    expect(await embeddedStore.get(operations.simple.hash)).toMatchObject(
+    expect(documentStore.opts.store['cache'].calculatedSize).toBe(508);
```

Comment:
This changed because the underlying store is wrapping the cached item like so, making it slightly larger:
`{ value: <item>, expires: <expiration value>}`